webv2/updateTableSchema/removeColumn.go

```diff
@@ -0,0 +1,331 @@
+}
+
+// removeColumnFromForeignkeyReferTableSchema remove given column from Foreign key Refer Table Schema.
+func removeColumnFromForeignkeyReferTableSchema(Conv *internal.Conv, referTable ddl.CreateTable, table string, colName string) {
+
+	referTable = removeColumnFromSpannerColDefs(referTable, colName)
+
+	// step II
+	referTable = removeColumnFromSpannerPK(referTable, colName)
+
+	// step III
+	referTable = removeColumnFromSpannerSecondaryIndex(referTable, colName)
+
+	// step IV
+	referTable = removeColumnFromSpannerForeignkeyColumns(referTable, colName)
+
+	// step V
+	referTable = removeColumnFromSpannerForeignkeyReferColumns(referTable, colName)
+
+	// step VI
+	referTable = removeColumnFromSpannerColNames(referTable, colName)
+
+	// step VII
+	removeSpannerSchemaIssue(table, colName, Conv)
+
+	// step VIII
+	removeColumnFromToSpannerToSource(table, colName, Conv)
+
+	Conv.SpSchema[table] = referTable
+
+}
+
+// removeColumnFromParentTableSchema remove given column from interleaved Parent Table Schema.
+func removeColumnFromParentTableSchema(Conv *internal.Conv, parentSchemaTable string, colName string) {
+
+	childSchemaSp := Conv.SpSchema[parentSchemaTable]
+
+	childSchemaSp = removeColumnFromSpannerColNames(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerColDefs(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerPK(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerSecondaryIndex(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerForeignkeyColumns(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerForeignkeyReferColumns(childSchemaSp, colName)
+
+	removeSpannerSchemaIssue(parentSchemaTable, colName, Conv)
+	removeColumnFromToSpannerToSource(parentSchemaTable, colName, Conv)
+
+	Conv.SpSchema[parentSchemaTable] = childSchemaSp
+
+}
+
+// removeColumnFromChildTableSchema remove given column from interleaved Child Table Schema.
+func removeColumnFromChildTableSchema(Conv *internal.Conv, childSchemaTable string, colName string) {
+
+	childSchemaSp := Conv.SpSchema[childSchemaTable]
+
+	childSchemaSp = removeColumnFromSpannerColNames(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerColDefs(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerPK(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerSecondaryIndex(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerForeignkeyColumns(childSchemaSp, colName)
+	childSchemaSp = removeColumnFromSpannerForeignkeyReferColumns(childSchemaSp, colName)
+
+	removeSpannerSchemaIssue(childSchemaTable, colName, Conv)
+	removeColumnFromToSpannerToSource(childSchemaTable, colName, Conv)
+
+	Conv.SpSchema[childSchemaTable] = childSchemaSp
+
+}
+
+// removeColumnFromSpannerColNames remove given column from ColNames.
+func removeColumnFromSpannerColNames(sp ddl.CreateTable, colName string) ddl.CreateTable {
+
+	// step VI
+	// remove sp.ColNames
```

Comment:
We can remove these comments from inside the methods. Same for all the methods below

Reply:
Removed redundant comments